assets/dev/scss/frontend/widgets/spacer.scss

```diff
@@ -28,5 +29,23 @@
 				}
 			}
 		}
+
+		.elementor-widget-empty-icon {
+			position: absolute;
+			top: 0;
+			bottom: 0;
+			left: 0;
+			right: 0;
+			height: 22px;
+			margin: auto 0 !important;
```

Comment:
What do we need !important for?

Alternatively try to remove important and duplicate the class name (like .elementor-widget-empty-icon.elementor-widget-empty-icon).